packages/roosterjs-editor-api/lib/format/clearFormat.ts

```diff
@@ -6,7 +6,11 @@ import setTextColor from './setTextColor';
 import { Editor } from 'roosterjs-editor-core';
 import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
 
-export default function clearFormat(editor: Editor): void {
+/**
+ * Clear the format in current selection
```

Comment:
Need to mention what's the result format state after clearing: it will be changed to default format.

Reply:
Also mention which format will be changed. (B/I/U/font name/font size/...) You need to try in different browsers.